index.css

```
@@ -0,0 +1,171 @@
+html body {
+    font-family: Helvetica, sans-serif;
+    font-size: 22px;
+    margin: 0;
+    padding: 0;
+    background-color: black;
+    color: rgba(255, 246, 240, 0.9);
+}
+
+nav {
+    display: inline-flex;
+    position: fixed;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: solid rgb(255, 245, 238, 0.9) 1px;
+    height: 69px;
+    width: 100%;
+    background-color: black;
+    top: 0;
+}
+
+nav li {
+    display: inline-block;
+    align-content: center;
+}
+
+nav img {
+    height: 50px;
+    margin-left: 10px;
+}
+
+nav a {
+    color: rgb(255, 245, 238, 0.9);
```

```suggestion
  padding: 10px;
```

A better use for padding. Increases the hitspace for the link